tests/framework/db/mssql/QueryBuilderTest.php

```
@@ -628,7 +624,9 @@ public function testDropColumnDropsConstraintOnDb(
         string $table,
         string $column,
     ): void {
-        $db = $this->getConnection(false);
+        $db = $this->getConnection(false, false);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)